model/src/w3parall.F90

```diff
@@ -93,8 +93,16 @@ MODULE W3PARALL
   REAL,  PARAMETER     :: ONETHIRD  = 1.0d0/3.0d0
   REAL,  PARAMETER     :: ZERO      = 0.0d0
 
-  REAL*8,  PARAMETER     :: THR8      = TINY(1.d0)
-  REAL,  PARAMETER     :: THR       = TINY(1.0)
+  REAL*8,  PARAMETER    :: THR8 = TINY(1.d0)
+  REAL,  PARAMETER      :: THR  = TINY(1.0)
+
+  REAL*8, PARAMETER     :: PI = 3.14159265358979323846D0
```

Comment:
These variables are defined in model/src/constants.F90, can we simply just use those instead of having the same variable defined in multiple places?